.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
I didn't think comments are valid in JSON

Reply:
vscode made this file so I guess it works for them